mobile/src/main/java/org/openhab/habdroid/ui/LogActivity.java

```diff
@@ -127,6 +129,7 @@ protected String doInBackground(Boolean... clear) {
                     logBuilder.append(line);
                     logBuilder.append(separator);
                 }
+                bufferedReader.close();
```

Comment:
The reader won't be closed if an exception is thrown. Please see https://lgtm.com/rules/10030075/ for the correct way to fix this issue.

Reply:
Thank you for the fix! I implemented as described in the rules.